FEM/fem_ele.cpp

```diff
@@ -1236,9 +1422,10 @@ double CElement::CalcXi_p()
 		}
 		r = 1.0 / Xi_p;
 		Xi_p = r;
+		return Xi_p;
 	}
-
-	return Xi_p;
+	else
+		return 0.;
```

Comment:
If compare line 1427 and 1428, you will see that the indentation is OK. Line 1427 has an indentation of one tab, while line 1428 has two tabs of the indentation.  It looks like that one tab covers 8 charter space on this page.

I will apply clang-format once this PR is merged.